src/main/java/telraam/monitoring/StationDetectionManager.java

```diff
@@ -0,0 +1,32 @@
+package telraam.monitoring;
+
+import telraam.database.daos.DetectionDAO;
+import telraam.database.daos.TeamDAO;
+import telraam.database.models.Detection;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class StationDetectionManager {
+    private DetectionDAO detectionDAO;
+
+    public StationDetectionManager(DetectionDAO detectionDAO) {
+        this.detectionDAO = detectionDAO;
+    }
+
+    public Map<Integer, Long> timeSinceLastDetectionPerStation() {
+        Set<Integer> stationIdList = this.detectionDAO.getAll().stream().map(Detection::getStationId).collect(Collectors.toSet());
```

Comment:
This is not the way to go, This will nuke our database and will cause performance issues in telraam with this endpoint being called rapidly in grafana